javascript/app.js

```diff
@@ -42,11 +42,15 @@ FinancialFreedom.directive('autofocus', ['$timeout', function($timeout) {
   }
 }]);
 
-FinancialFreedom.controller('bodyController', ['$scope', '$location',  function($scope, $location) {
+FinancialFreedom.controller('bodyController', ['$scope', '$location', '$window', function($scope, $location, $window) {
     $location.path("/");
     $scope.isActive = function(route) {
         return route == $location.path();
     };
+
+    $scope.$on('$routeChangeSuccess', function(event) {
+        $window.ga('send', 'pageview', { page: $location.url() });
+    });

```

Comment:
This is fine for now, but ultimately it would be cool if we created a GoogleAnalytics service so we can just include the service and call a simple method on that service to report different things to Google. Ex: `GoogleAnalyticsService.reportPageView(url)`